package.json

```diff
@@ -19,7 +19,8 @@
     "build:webpack": "webpack --progress --colors --config build/webpack.dist.babel.js",
     "build:watch": "webpack --progress --colors --config build/webpack.dist.babel.js --watch",
     "test": "npm run karma",
-    "karma": "cross-env NODE_ENV=test babel-node node_modules/karma/bin/karma start build/karma.conf.js --reporters nyan,coverage --single-run",
+    "coverage": "cross-env NODE_ENV=test babel-node node_modules/karma/bin/karma start build/karma.conf.js --reporters nyan,coverage --single-run",
+    "karma": "cross-env NODE_ENV=test babel-node node_modules/karma/bin/karma start build/karma.conf.js --reporters spec --single-run",
```

Comment:
I think we don't need to decline all combinations of commands.
The CI can run `yarn karma --reporters [reporters we want]`.